R/smtp_send.R

```diff
@@ -291,13 +291,15 @@ smtp_send <- function(email,
     send_result <-
       processx::run(
         command = binary_loc,
-        args = run_args
+        args = run_args,
+        error_on_status = FALSE
       )
 
     if (send_result$status == 0) {
       message("The email message was sent successfully.\n")
     } else {
       message("The email message was NOT successfully sent.\n")
+      message(send_result$stderr)
```

Comment:
Note to self: Lines 301 and 302 should be `warning()`s and a final `stop()` call should be added.